# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1325869742 18000
#      Fri Jan 06 12:09:02 2012 -0500
# Branch stable
# Node ID bb5790c5e713056b3da3241129a0543cd4a21ec5
# Parent  727504fd26f0b940cc352389f7aa8d5204c2222f
maint: update gnulib

diff --git a/.hgsubstate b/.hgsubstate
--- a/.hgsubstate
+++ b/.hgsubstate
@@ -1,1 +1,1 @@
-bb052d4a7416accaad0747e84bd2a0accbfcf923 gnulib
+3559997a03e82d650aaa708447362fe20a6eaf15 gnulib
